cmd/etcd.go

```diff
@@ -14,20 +14,46 @@ import (
 	"github.com/urfave/cli"
 )
 
+const s3Endpoint = "s3.amazonaws.com"
+
 func EtcdCommand() cli.Command {
 	snapshotFlags := []cli.Flag{
 		cli.StringFlag{
 			Name:  "name",
-			Usage: "Specify Snapshot name",
+			Usage: "Specify snapshot name",
 		},
 		cli.StringFlag{
 			Name:   "config",
 			Usage:  "Specify an alternate cluster YAML file",
 			Value:  pki.ClusterConfig,
 			EnvVar: "RKE_CONFIG",
 		},
+		cli.BoolFlag{
+			Name:  "s3",
+			Usage: "Enabled backup to s3, set true or false",
+		},
+		cli.StringFlag{
+			Name:  "endpoint",
```

Comment:
Do you think we should prepend the s3 related options with `s3` ?

Reply:
@alena1108 and you have diffrent opion of the `endpoint` name, but I would prefer to go with the name of `s3-endpoint`.